diff --git a/build.zig b/build.zig
index 89142561b..5d4ab82ab 100644
--- a/build.zig
+++ b/build.zig
@@ -155,16 +155,16 @@ pub fn build(b: *std.Build) !void {
     // libghostty-vt xcframework (Apple only, universal binary).
     // Only when building on macOS (not cross-compiling) since
     // xcodebuild is required.
-    if (builtin.os.tag.isDarwin() and config.target.result.os.tag.isDarwin()) {
-        const apple_libs = try buildpkg.GhosttyLibVt.initStaticAppleUniversal(
-            b,
-            &config,
-            &deps,
-            &mod,
-        );
-        const xcframework = buildpkg.GhosttyLibVt.xcframework(&apple_libs, b);
-        b.getInstallStep().dependOn(xcframework.step);
-    }
+//    if (builtin.os.tag.isDarwin() and config.target.result.os.tag.isDarwin()) {
+//        const apple_libs = try buildpkg.GhosttyLibVt.initStaticAppleUniversal(
+//            b,
+//            &config,
+//            &deps,
+//            &mod,
+//        );
+//        const xcframework = buildpkg.GhosttyLibVt.xcframework(&apple_libs, b);
+//        b.getInstallStep().dependOn(xcframework.step);
+//    }
 
     // Helpgen
     if (config.emit_helpgen) deps.help_strings.install();
